libproxy/cmake/devfiles.cmk

```diff
@@ -1,4 +1,4 @@
-if(NOT WIN32)
+if(NOT WIN32 OR MINGW)
```

Comment:
I don't see this option here, probably only when I trigger new pull-request.
But ghub should support it on you side ... if enabled, you should see squash&merge